src/Containers/LeftSideBar/ReceiptElements.ts

```diff
@@ -0,0 +1,307 @@
+export interface IDraggableComponent {
+    [key: string]: {
+        key: string,
+        field: string,
+        isRequired: boolean,
+        isRecommended: boolean
+    }
+};
+
+export interface ICategoryInterface {
+    key: string,
+    editorCategory: string,
+    draggableComponents: IDraggableComponent
+};
+
+export interface ILeftSideBarInterface {
+    textElements: ICategoryInterface,
+    imageElements: ICategoryInterface,
+    layoutElements: ICategoryInterface,
+    numberElements: ICategoryInterface,
+    priceElements: ICategoryInterface,
+    codesElements: ICategoryInterface,
+    settingsElements: ICategoryInterface
+};
+
+export const ReceiptElements: ILeftSideBarInterface = {
```

Comment:
Why is this not in storybook, what if another developer has a different use case.